medicines/api/src/substance.rs

```diff
@@ -1,14 +1,16 @@
 use crate::{pagination, pagination::PageInfo, product::Product};
+use search_client::{models::IndexResults, Search};
+use std::collections::BTreeMap;
 
-#[derive(juniper::GraphQLObject)]
+#[derive(Debug, juniper::GraphQLObject)]
 #[graphql(description = "An active ingredient found in medical products")]
 pub struct Substance {
     name: String,
-    products: Option<Vec<Product>>,
+    products: Vec<Product>,
```

Comment:
I think by doing this you make `product` non-optional on Substance in the graphQL